packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -6,6 +6,74 @@ import 'package:flutter/widgets.dart';
 import 'bottom_tab_bar.dart';
 import 'theme.dart';
 
+/// Coordinates tab selection between a [CupertinoTabBar] and a [CupertinoTabScaffold].
+///
+/// The [currentIndex] property is the index of the selected tab, changing its value updates
```

Comment:
I'd just call this property [index] like the Material TabController. 'current' doesn't add much value.

Reply:
Period after 'selected tab'. It's 2 sentences.